build/ci-build.yml

```diff
@@ -31,6 +27,30 @@ stages:
         pool:
           vmImage: 'ubuntu-16.04'
         steps:
+          - powershell: |
```

Comment:
Can we use this task? https://github.com/arcus-azure/azure-devops-templates/blob/master/nuget/determine-pr-version.yml